src/Simulation/Simulators.Tests/Circuits/TeleportTest.qs

```diff
@@ -7,8 +7,7 @@ namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits {
 
     @Test("QuantumSimulator")
     @Test("SparseSimulator")
-    // TODO: Disabled until we have a noise model for Rz.
-    // @Test("Microsoft.Quantum.Experimental.OpenSystemsSimulator")
+    @Test("Microsoft.Quantum.Experimental.OpenSystemsSimulator")
     operation TestTeleport() : Unit {
```

Comment:
I assume that some time later there will be a change in the Q# compiler such that the following change below will be understood by the compiler, similar to lines 8 - 9 above. If that's true then either a TODO comment here or a task wouldn't harm, to not forget to make a change in the compiler.
```suggestion
    @Test("OpenSystemsSimulator")
    operation TestTeleport() : Unit {
```
@bettinaheim, FYI.

Reply:
Yes, at this point cross-repo changes are difficult to test with end-to-end build issues, such that I didn't want to take that change as well with this PR. Definitely would be good to do, though, as we remove `Microsoft.Quantum.Experimental` from other parts of this feature.